advanced/Scripts/updatecheck.sh

```diff
@@ -41,54 +41,78 @@ function get_local_version() {
 # shellcheck disable=SC1091
 . /etc/pihole/setupVars.conf
 
+utilsfile="/opt/pihole/utils.sh"
+source "${utilsfile}"
```

Comment:
```suggestion
source "${utilsfile:-/opt/pihole/utils.sh}"
```

Reply:
I guess unless `utilsfile` is used elsewhere in the script.

Reply:
Heh, I actually just took this from another place where we source it (webpage.sh) but it certainly can stand to be changed